src/home/index.js

```diff
@@ -0,0 +1,12 @@
+import './make-user.js';
+import api from '../services/api.js';
+
+const userSignUp = document.getElementById('form');
+
+userSignUp.addEventListener('play', (event) => {
```

Comment:
Change 'play' to 'submit' this line should look like 
`userSignUp.addEventListener('submit', (event) => {`